packages/terra-form-input/src/InputField.jsx

```diff
@@ -145,7 +140,6 @@ const InputField = (props) => {
     labelAttrs,
     maxWidth,
     onChange,
-    placeholder,
```

Comment:
Should placeholder also be deleted from the `customProps` if we outright disallow it? Otherwise, it would get spread onto FormField.

Reply:
Update here https://github.com/cerner/terra-core/pull/3075/commits/ad81efd1977642f263c01c46fda0a8f65f4ea6ba.